### DISCUSSION TITLE
Can spanmetricsconnector be used to collect metrics from unsampled traces/spans

### DISCUSSION THREAD

**Discussion by MaheshGPai** [open-telemetry/opentelemetry-collector-contrib#32842] [Q&A]
If `spanmetricsconnector` is used along with `tailsamplingprocessor`, it will not collect the metrics of spans/traces dropped by the processor. Is there a way to collect the metrics even for dropped spans.
I believe this was earlier a processor - so one could technically configure this prior to the `tailsamplingprocessor` and collect high fidelity metrics. But with this getting converted connector, we may be loosing this capability.
Can you please confirm or correct my understanding?

Thanks & Regards,
Mahesh

**Comment by kes2464** [comment]
I am having exactly the same concern. Could someone please confirm?
